pkg/helm/helmtest/helmtest.go

```diff
@@ -73,15 +75,26 @@ func (e *NamespacedEnv) Ctl() *kube.Ctl {
 	return e.env.Ctl()
 }
 
-func (e *NamespacedEnv) Install(chart string, opts helm.InstallOptions) helm.Release {
-	require.Zero(e.t, opts.Name, ".Name may not be specified")
-	require.Zero(e.t, opts.Namespace, ".Namespace may not be specified")
+func (e *NamespacedEnv) Install(ctx context.Context, chart string, opts helm.InstallOptions) helm.Release {
 	require.False(e.t, opts.CreateNamespace, ".CreateNamespace may not be specified")
 
 	opts.Namespace = e.namespace.Name
 
-	release, err := e.env.helm.Install(context.Background(), chart, opts)
+	// Every time, Redpanda chart is installed, the next deployment would fail
```

Comment:
I would argue that this mutex should be used in the actual tests (Or we should disable the node port service in our tests). This struct is completely safe for concurrent use, so let's not limit it's abilities because a user thereof is not.

Reply:
Ok, I will remove mutex and external connectivity properties in values (node ports).